rasa_nlu/config.py

```diff
@@ -32,7 +32,7 @@
     "max_number_of_ngrams": 7,
     "pipeline": [],
     "response_log": "logs",
-    "duckling_processing_mode": "append",
+    "duckling_dimensions": [],
```

Comment:
It's actually all modules. I could also import the list from the duckling module to make it more explicit, but then if the duckling module ever imports `config` then we have a loop :curly_loop: